ui/src/router.tsx

```diff
@@ -96,3 +96,8 @@ function NotFound(props: { error: Error; resetErrorBoundary?: () => void }) {
     </>
   );
 }
+
+export const getCurrentPageUrl = (): string => {
```

Comment:
`export function getCurrentUrl(): string {` to be consistent with the rest of the code. Also drop `Page` from the name.